.env.sample

```diff
@@ -3,4 +3,5 @@ SCHEME=http
 # Random string (generated with openssl rand -base64 32)
 ROCKET_SECRET_KEY=Qt6DPFUU8qO4BKTCQnKAgt9FBBJxIWAYUGyHuruVfpE=
 ROCKET_LOG=normal
-ROCKET_DATABASES={service_cache={url="redis://127.0.0.1:6379"}}
\ No newline at end of file
+ROCKET_DATABASES={service_cache={url="redis://127.0.0.1:6379"}}
+WEBHOOK_TOKEN=some_random_token
```

Comment:
Maybe it would be nice to mention something about this new env values in the README. Can be forked to a different issue though. Maybe for 1.0.0

Reply:
Yes we need to adjust our docs for 1.0.0